README.md

```diff
@@ -229,12 +229,19 @@ New statuses are up to the commissioners' discretion until the league votes on t
 - The winner of the Toilet Bowl must wear the toilet seat at the draft and pay for the championship trophy plaque
 
 ### Golf ###
-- Takes place Championship week (Week 17)
+- Takes place Championship week for teams not participating in the championship (Week 17)
 - The lowest score wins the $20 weekly prize
 **Rules**
 - If a player doesn't play any snaps, then the owner is disqualified
 - If a player doesn't score any points but plays a snap, then they receive 20 points
-- A player is classified as scoring points if, at any point during the game, their total points change (not zero)
+- Player must do at least one point scoring action (positive or negative points)
+- Examples
+  	- Plays that count:
+  	  	- Any touch that becomes yards (ex: 1 carry for -1 yards & 1 carry for 1 yard (2 carries for total zero yards, 1 catch for -5 yards)
+  	  	- Fumbles
+  	- Plays that don't count
+  		- Targets
+  	   	- Carry for zero yards
```

Comment:
does this not count? I thought it would and that they had to touch the ball basically

Reply:
fixed based off poll in discord